sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 
 ### Features Added
 
+- Add support for enabling idempotent partitions using `EventHubBufferedProducerClient`.
```

Comment:
I guess since the new feature is for advanced scenarios, it does not have to be mentioned in the README.md, no? /cc @ramya-rao-a

Reply:
.NET doesn't mention anything about the buffered producer in README. Will keep the consistency :)